docs/announce/6.e.md

```diff
@@ -32,10 +32,10 @@ This ChangeLog is targeted towards language users, to help with preparation to u
 compilers supporting latest language version. Thus, it does not contain every minute
 change to the specification that occurred. Implementations wishing to ensure full
 compliance with the new version of the language specification should execute the
-test suite available at https://github.com/perl6/roast/ and examine any failing tests.
+test suite available at https://github.com/raku/roast/ and examine any failing tests.
 
```

Comment:
Again, capitalizing is probably better. It's redirected, but still it's better to use like it should.

Reply:
Fixed the capitalization. Thanks!